base_location_geonames_import/models/country.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Nicolas Bessi, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCountryState(models.Model):
+
+    _inherit = 'res.country'
+
+    geonames_state_name_column = fields.Integer(
```

Comment:
Remove `name` part as it makes the field name very long and doesn't add a special meaning

Reply:
ok, what do u think of geonames_state_name and geonames_state_code, I've removed the _column that was redundant and not helpfull

Reply:
I prefer `geonames_state_column`, which is more relevant.

Reply:
Wait, I didn't remember about the code/name duality. OK then